components/zcash/rs/Cargo.toml

```diff
@@ -0,0 +1,14 @@
+[package]
+name = "zcash-cxx"
+version = "1.0.0"
+edition = "2021"
+license = "MPL-2.0"
+
+[dependencies]
+cxx = { version = "1.0", features = [ "alloc", "std" ] }
+orchard = { version = "0.7.1",  default-features = false }
```

Comment:
this is currently disabling `multicore` which is good in that it eliminates the thread pool, but I think we need to deal with the thread pool in another way here since I understand that perf will be very poor and this compounds the issue of running things on the UI thread  mentioned above

Reply:
I guess this is ok for now because it only affects `orchard::builder`

Reply:
This will decrease performance specifically of transaction creation (the `Bundle::create_proof` method on the output of `Builder::build`). No other functionality in the `orchard` crate uses `multicore`.